incubator/kafka/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: Apache Kafka is publish-subscribe messaging rethought as a distributed
   commit log.
 name: kafka
-version: 0.2.13
+version: 0.3.0
```

Comment:
Bumped minor version because some of the top-level config values nested underneath `persistence`. Otherwise, could be `0.2.14`